src/types/line.js

```diff
@@ -157,12 +157,26 @@ module.exports = function(Chart) {
 
 			ctx.font = chartHelpers.fontString(model.labelFontSize, model.labelFontStyle, model.labelFontFamily);
 			var textWidth = ctx.measureText(model.labelContent).width;
-			var textHeight = ctx.measureText('M').width;
+			var textHeight = model.labelFontSize;
+			model.labelHeight = textHeight + (2 * model.labelYPadding);
+
+			if (model.labelContent !== null && model.labelContent.indexOf('\n') !== -1) {
```

Comment:
This seems like it would fail if `labelContent` is `undefined`. Maybe just check `model.labelContent` rather than `model.labelContent !== null`